server/auth.go

```diff
@@ -441,11 +425,6 @@ func (s *Server) processClientOrLeafAuthentication(c *client, opts *Options) boo
 		}
 	} else {
 		tlsMap = opts.LeafNode.TLSMap
-		tlsPinnedCerts = opts.LeafNode.TLSPinnedCerts
-	}
-	if !s.matchesPinnedCert(c, tlsPinnedCerts) {
-		s.mu.Unlock()
-		return false
 	}
 	if !authRequired {
```

Comment:
Would you mind then moving this back to where it was prior to the first PR you made about pinned_certs?

Reply:
fixed